web/src/components/Wrapper.js

```diff
@@ -17,7 +20,13 @@ class Wrapper extends Component {
 
   render() {
     const { children } = this.props;
-    return <div className="site">{children}</div>;
+    let showSiteTitle = location.pathname === '/'; // eslint-disable-line prefer-const
+    return (
+      <div className="site">
+        <Header key={location.pathname} showSiteTitle={showSiteTitle} />
```

Comment:
I'm not sure what the `key` prop is for here.  It's not listed in the propTypes for the Header.

Reply:
Oh, whoops! I'll add it. The key forces the `Header` to rerender when `location.pathname` changes; otherwise it doesn't consistently update itself on navigation. (Mostly I was using this to force `ariaExpanded` to reset itself when the user navigated to a page from the dropdown menu... maybe I could just add a handler to those items... 🤔 )